### PR TITLE
Update con-comparing-groups-roles.adoc

### PR DESCRIPTION
**Pull request by ciis0**
add missing comma. :)

### CODE REVIEW

File: docs/documentation/server_admin/topics/roles-groups/con-comparing-groups-roles.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 = Groups compared to roles
 [role="_abstract"]
-Groups and roles have some similarities and differences. In {project_name}, groups are a collection of users to which you apply roles and attributes. Roles define types of users and applications assign permissions and access control to roles.
+Groups and roles have some similarities and differences. In {project_name}, groups are a collection of users to which you apply roles and attributes. Roles define types of users and applications, assign permissions and access control to roles.
```
Comment (ciis0) [inline docs/documentation/server_admin/topics/roles-groups/con-comparing-groups-roles.adoc:5]: also, is "roles define [...] access control to roles" intentional?

Comment (ahus1) [reply docs/documentation/server_admin/topics/roles-groups/con-comparing-groups-roles.adoc:5]: I'd think think the comma goes here: 

```suggestion
Groups and roles have some similarities and differences. In {project_name}, groups are a collection of users to which you apply roles and attributes. Roles define types of users, and applications assign permissions and access control to roles.
```

**Review by ahus1 — CHANGES_REQUESTED** [review]
Thank you for this contribution! 

To contribute to this project, you need to sign-off your commits as outlined in the `CONTRIBUTING.md`.

So please add a 

```
Signed-off-by: ...
```

to your commit. Thanks!

**Comment by ciis0** [comment]
oops, messed up the commits due to a shallow clone, will open a new PR.

**Pull request closed without merging**